TrafficCapture/trafficReplayer/src/test/java/org/opensearch/migrations/replay/kafka/KafkaTrafficCaptureSourceLongTermTest.java

```diff
@@ -61,7 +61,7 @@ public void testTrafficCaptureSource() throws Exception {
         for (int i = 0; i < TEST_RECORD_COUNT; ) {
             Thread.sleep(getSleepAmountMsForProducerRun(i));
             var nextChunkFuture = kafkaTrafficCaptureSource.readNextTrafficStreamChunk(rootContext::createReadChunkContext);
-            var recordsList = nextChunkFuture.get((2 + TEST_RECORD_COUNT) * PRODUCER_SLEEP_INTERVAL_MS, TimeUnit.MILLISECONDS);
+            var recordsList = nextChunkFuture.get((2 * TEST_RECORD_COUNT) * PRODUCER_SLEEP_INTERVAL_MS, TimeUnit.MILLISECONDS);
```

Comment:
Encountered a few failures when this was not long enough and running other unit tests in parallel, adjusted the buffer to 2x time.